server/server_event.py

```diff
@@ -4,19 +4,27 @@
     notify_error_to_client,
     notify_your_turn,
     notify_user_list_to_client,
+    notify_end_game_to_client,
+)
+from server.web_requests import (
+    notify_end_game_to_web,
 )
-# from server.web_requests import notify_game_created
 from server.exception import GameIdException
 from server.grpc_adapter import GRPCAdapterFactory
+from server.constants import (
+    GAME_STATE_ACCEPTED,
+    GAME_STATE_ENDED,
+    LAST_PLAYER,
+)
 
 
 class ServerEvent(object):
     def __init__(self, response, client):
         self.response = response
         self.client = client
 
-    # def run(self):
-    #     raise NotImplementedError
+    def run(self):
+        raise NotImplementedError
```

Comment:
It's a good practice to log some info before raising an exception, so you're able to catch the stacktrace and have more information to debug in case of an error